### ISSUE TITLE
PI levels fiesta

### ISSUE THREAD

**Issue by myleneb** [CaliOpen/Caliopen#495]
`message.pi` seem to send different values, depending if you call the discussion or the message:

Response when displaying /discussion : 
```
{"total": 1,"messages":[{"attachments":null
,"body":"\n\n\n\n\n\n\n\u003cp\u003e\n\u003ca href=\"http://app.trouver-presta.fr/v/?camp=9544767838567738696_8\u0026amp;ms=bGF1cmVudEBicmFpbnN0b3JtLmZy\" title=\"Si cet e-mail ne s\u0026#39;affiche pas correctement, suivez ce lien.\" rel=\"nofollow\"\u003eSi cet email ne s’affiche pas correctement, suivez ce lien.\u003c/a\u003e\u003c/p\u003e\u003ctable width=\"600\"\u003e     \u003ctbody\u003e         \u003ctr\u003e             \u003ctd width=\"600\" height=\"40\" align=\"center\" valign=\"middle\"\u003eMa Nouvelle Caisse enregistreuse est un jeu d\u0026#39;enfant.\u003cbr\u003e             Si ce message ne s\u0026#39;affiche pas correctement, cliquez ici.\u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd width=\"600\" height=\"77\" align=\"center\"\u003e\u003cimg src=\"http://app.trouver-presta.fr/r/?rc=aHR0cHM6Ly9ub2Rlcy5uZW9wZXJmLmNvbS91cGxvYWRzL3Zpc3VlbHMvMTMvMjcva2l0cy9Mb2dvX0Jwcm8uanBn\" alt=\"BPRO\" width=\"136\" height=\"77\"\u003e\u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd width=\"600\" height=\"43\" align=\"center\" valign=\"middle\"\u003eCaisse enregistreuse\u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd width=\"600\" height=\"2\"\u003e \u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd\u003e             \u003ctable width=\"600\"\u003e                 \u003ctbody\u003e                     \u003ctr\u003e                         \u003ctd width=\"72\" height=\"15\"\u003e \u003c/td\u003e                         \u003ctd width=\"1\" height=\"15\"\u003e \u003c/td\u003e                         \u003ctd width=\"66\" height=\"15\"\u003e \u003c/td\u003e                         \u003ctd width=\"419\" height=\"15\"\u003e \u003c/td\u003e                         \u003ctd height=\"15\"\u003e \u003c/td\u003e                     \u003c/tr\u003e                 \u003c/tbody\u003e             \u003c/table\u003e             \u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd\u003e             \u003ctable width=\"600\"\u003e                 \u003ctbody\u003e                     \u003ctr\u003e                         \u003ctd\u003e\u003ca href=\"http://app.trouver-presta.fr/r/?m=bGF1cmVudEBicmFpbnN0b3JtLmZy\u0026amp;c=9544767838567738696_8\u0026amp;rc=aHR0cDovL25vZGVzLm5wdjE3cWVqYmQuY29tL3IvY2xpYy0yNy0zNjItMzAxOC11cmwtYUhSMGNEb3ZMMkp3Y204dVpuSXZNemN5TUY5Q1VETmZUbVZ2Y0Q5eFkzQTlNemN5TUY5Q1VETmZUbVZ2Y0NOMWRHMWZjMjkxY21ObFBXNG1kWFJ0WDIxbFpHbDFiVDFsTFcxaGFXd21kWFJ0WDJOaGJYQmhhV2R1UFc1bGQzTmpKblYwYlY5dWIyOTJaWEp5YVdSbFBURT9yZ3JvdXA9cF8xMTM1MSZhbXA7dHJhY2thZmY9JmFtcDtncm91cD0=\" rel=\"nofollow\"\u003e\u003cimg src=\"http://app.trouver-presta.fr/r/?rc=aHR0cHM6Ly9ub2Rlcy5uZW9wZXJmLmNvbS91cGxvYWRzL3Zpc3VlbHMvMTMvMjcva2l0cy9QaWN0b19FcXVpcG10LmpwZw==\" width=\"139\" height=\"98\" alt=\"\"\u003e\u003c/a\u003e\u003c/td\u003e                         \u003ctd\u003e       \n                   \u003ctable width=\"419\"\u003e                             \u003ctbody\u003e                                 \u003ctr\u003e                                     \u003ctd width=\"19\"\u003e \u003c/td\u003e                                     \u003ctd width=\"400\" height=\"4\"\u003e \u003c/td\u003e                                 \u003c/tr\u003e                                 \u003ctr\u003e                                     \u003ctd width=\"19\" height=\"60\"\u003e \u003c/td\u003e                                     \u003ctd width=\"400\" align=\"left\" valign=\"middle\"\u003e\u003cspan\u003eAvec ma nouvelle caisse enregistreuse\u003cbr\u003e                                     \u003c/span\u003el\u0026#39;encaissement est un jeu d\u0026#39;enfant !                 \u003c/td\u003e                                 \u003c/tr\u003e                                 \u003ctr\u003e                                     \u003ctd width=\"19\"\u003e \u003c/td\u003e                                     \u003ctd width=\"400\" height=\"4\"\u003e \u003c/td\u003e                                 \u003c/tr\u003e                             \u003c/tbody\u003e                         \u003c/table\u003e                         \u003c/td\u003e                         \u003ctd width=\"42\" height=\"98\"\u003e \u003c/td\u003e                     \u003c/tr\u003e                 \u003c/tbody\u003e             \u003c/table\u003e             \u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd\u003e             \u003ctable width=\"600\"\u003e           \n       \u003ctbody\u003e                     \u003ctr\u003e                         \u003ctd width=\"72\" height=\"53\"\u003e \u003c/td\u003e                         \u003ctd width=\"1\"\u003e \u003c/td\u003e                         \u003ctd width=\"66\"\u003e \u003c/td\u003e                         \u003ctd width=\"419\"\u003e\u003ca href=\"http://app.trouver-presta.fr/r/?m=bGF1cmVudEBicmFpbnN0b3JtLmZy\u0026amp;c=9544767838567738696_8\u0026amp;rc=aHR0cDovL25vZGVzLm5wdjE3cWVqYmQuY29tL3IvY2xpYy0yNy0zNjItMzAxOC11cmwtYUhSMGNEb3ZMMkp3Y204dVpuSXZNemN5TUY5Q1VETmZUbVZ2Y0Q5eFkzQTlNemN5TUY5Q1VETmZUbVZ2Y0NOMWRHMWZjMjkxY21ObFBXNG1kWFJ0WDIxbFpHbDFiVDFsTFcxaGFXd21kWFJ0WDJOaGJYQmhhV2R1UFc1bGQzTmpKblYwYlY5dWIyOTJaWEp5YVdSbFBURT9yZ3JvdXA9cF8xMTM1MSZhbXA7dHJhY2thZmY9JmFtcDtncm91cD0=\" rel=\"nofollow\"\u003e                         \u003cul\u003e                             \u003cli\u003eFacile\u003c/li\u003e                             \u003cli\u003ePratique\u003c/li\u003e                             \u003cli\u003eErgonomique\u003c/li\u003e   \n                       \u003c/ul\u003e                         \u003c/a\u003e\u003c/td\u003e                         \u003ctd width=\"42\"\u003e \u003c/td\u003e                     \u003c/tr\u003e                 \u003c/tbody\u003e             \u003c/table\u003e             \u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd\u003e             \u003ctable width=\"600\"\u003e                 \u003ctbody\u003e                     \u003ctr\u003e                         \u003ctd width=\"72\" height=\"16\"\u003e \u003c/td\u003e                         \u003ctd width=\"1\" height=\"16\"\u003e \u003c/td\u003e                                                  \u003ctd height=\"16\" width=\"485\"\u003e \u003c/td\u003e                                                  \u003ctd width=\"42\" height=\"16\"\u003e \u003c/td\u003e                     \u003c/tr\u003e                 \u003c/tbody\u003e             \u003c/table\u003e             \u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd\u003e             \u003ctable width=\"600\"\u003e                 \u003ctbody\u003e                     \u003ctr\u003e                         \u003ctd width=\"72\" height=\"178\"\u003e \u003c/td\u003e                         \u003ctd width=\"1\" height=\"178\"\u003e \u003c/td\u003e                         \u003ctd\u003e\u003ca href=\"http://app.trouver-presta.fr/r/?m=bGF1cmVudEBicmFpbnN0b3JtLmZy\u0026amp;c=9544767838567738696_8\u0026amp;rc=aHR0cDovL25vZGVzLm5wdjE3cWVqYmQuY29tL3IvY2xpYy0yNy0zNjItMzAxOC11cmwtYUhSMGNEb3ZMMkp3Y204dVpuSXZNemN5TUY5Q1VETmZUbVZ2Y0Q5eFkzQTlNemN5TUY5Q1VETmZUbVZ2Y0NOMWRHMWZjMjkxY21ObFBXNG1kWFJ0WDIxbFpHbDFiVDFsTFcxaGFXd21kWFJ0WDJOaGJYQmhhV2R1UFc1bGQzTmpKblYwYlY5dWIyOTJaWEp5YVdSbFBURT9yZ3JvdXA9cF8xMTM1MSZhbXA7dHJhY2thZmY9JmFtcDtncm91cD0=\" rel=\"nofollow\"\u003e\u003cimg src=\"http://app.trouver-presta.fr/r/?rc=aHR0cHM6Ly9ub2Rlcy5uZW9wZXJmLmNvbS91cGxvYWRzL3Zpc3VlbHMvMTMvMjcva2l0cy9WaXN1ZWwuanBn\" width=\"485\" height=\"178\" alt=\"\"\u003e\u003c/a\u003e\u003c/td\u003e                         \u003ctd width=\"42\" height=\"178\"\u003e \u003c/td\u003e                     \u003c/tr\u003e                 \u003c/tbody\u003e             \u003c/table\u003e                      \u003c/tr\u003e         \u003ctr\u003e             \u003ctd\u003e             \u003ctable width=\"600\"\u003e                 \u003ctbody\u003e                     \u003ctr\u003e                         \u003ctd width=\"72\" height=\"33\"\u003e \u003c/td\u003e                         \u003ctd width=\"1\" height=\"33\"\u003e \u003c/td\u003e                         \u003ctd width=\"527\" height=\"33\"\u003e \u003c/td\u003e                     \u003c/tr\u003e                 \u003c/tbody\u003e             \u003c/table\u003e             \u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd\u003e             \u003ctable width=\"600\"\u003e                 \u003ctbody\u003e                     \u003ctr\u003e                         \u003ctd width=\"72\" height=\"65\"\u003e \u003c/td\u003e                         \u003ctd width=\"1\" height=\"65\"\u003e \u003c/td\u003e                         \u003ctd width=\"91\" height=\"65\"\u003e \u003c/td\u003e                         \u003ctd align=\"center\"\u003e\u003ca href=\"http://app.trouver-presta.fr/r/?m=bGF1cmVudEBicmFpbnN0b3JtLmZy\u0026amp;c=9544767838567738696_8\u0026amp;rc=aHR0cDovL25vZGVzLm5wdjE3cWVqYmQuY29tL3IvY2xpYy0yNy0zNjItMzAxOC11cmwtYUhSMGNEb3ZMMkp3Y204dVpuSXZNemN5TUY5Q1VETmZUbVZ2Y0Q5eFkzQTlNemN5TUY5Q1VETmZUbVZ2Y0NOMWRHMWZjMjkxY21ObFBXNG1kWFJ0WDIxbFpHbDFiVDFsTFcxaGFXd21kWFJ0WDJOaGJYQmhhV2R1UFc1bGQzTmpKblYwYlY5dWIyOTJaWEp5YVdSbFBURT9yZ3JvdXA9cF8xMTM1MSZhbXA7dHJhY2thZmY9JmFtcDtncm91cD0=\" rel=\"nofollow\"\u003e\u003cimg src=\"http://app.trouver-presta.fr/r/?rc=aHR0cHM6Ly9ub2Rlcy5uZW9wZXJmLmNvbS91cGxvYWRzL3Zpc3VlbHMvMTMvMjcva2l0cy9CdG5fRXF1aXBtdC5qcGc=\" alt=\"en savoir plus\" width=\"272\" height=\"65\"\u003e\u003c/a\u003e\u003c/td\u003e                         \u003ctd width=\"164\" height=\"65\"\u003e \u003c/td\u003e                     \u003c/tr\u003e                 \u003c/tbody\u003e             \u003c/table\u003e             \u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd\u003e\u003cimg width=\"600\" height=\"33\" alt=\"\"\u003e\u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd width=\"600\" height=\"35\" align=\"center\"\u003e \u003c/td\u003e         \u003c/tr\u003e     \u003c/tbody\u003e \u003c/table\u003e\u003cimg alt=\"\" src=\"http://app.trouver-presta.fr/r/?rc=aHR0cDovL25vZGVzLm5wdjE3cWVqYmQuY29tL21haWxpbmctMjctMzYyLTMwMTg/cmdyb3VwPXBfMTEzNTEmYW1wO2dyb3VwPQ==\" height=\"1\" width=\"1\"\u003e\u003cimg src=\"http://app.trouver-presta.fr/t/?i=9544767838567738696_8\u0026amp;m=bGF1cmVudEBicmFpbnN0b3JtLmZy\u0026amp;url=http://app.trouver-presta.fr/images/blank.jpg\" alt=\"_pspacer5\"/\u003e\u003cp\u003e\n\u003ca href=\"http://app.trouver-presta.fr/d/?camp=9544767838567738696_8\u0026amp;ms=bGF1cmVudEBicmFpbnN0b3JtLmZy\" title=\"Ne plus recevoir d\u0026#39;informations de notre part\" rel=\"nofollow\"\u003ePour se désabonner : Suivez ce lien.\u003c/a\u003e\u003cbr/\u003eSi ce message vous a causé un quelconque dérangement, nous vous prions de nous en excuser.\n\u003c/p\u003e\n\n\n\n\n"
,"body_is_plain":false,"excerpt":"Ma Nouvelle Caisse enregistreuse est un jeu d'enfant. Si ce message ne s'affiche pas correctement, cliquez ici. Caisse enregistreuse Avec ma nouvelle caisse enregistreuse l'encaissement est un jeu …"
,"date":"2016-04-05T03:56:40.000Z","date_delete":"0001-01-01T00:00:00.000Z","date_insert":"2017-09-28T07:21:02.666Z","discussion_id":"15c5f51c-027d-4852-9eb0-f52eed7512b8"
,"external_references":{"ancestors_ids":null,"message_id":"7af37de14807c55ac99890f58e2d2aca@trouver-presta.fr","parent_id":""}
,"identities":null
,"importance_level":-2
,"is_answered":false
,"is_draft":false
,"is_unread":false
,"message_id":"732156e2-e969-4259-ac1e-831d7c8e32b9"
,"parent_id":""
,"participants":[{"address":"caisse@trouver-presta.fr","contact_ids":["d46a8028-13fa-4eb8-9f96-24157bc4413b"],"label":"","protocol":"email","type":"From"},{"address":"dev@caliopen.local","contact_ids":["abfa55df-9a47-4107-8a24-63752d8b7fd8"],"label":"","protocol":"email","type":"To"}]
,"privacy_features":{"ingress_server":"esmtp-b46.trouver-presta.fr","is_spam":"True","mail_emitter_mx_reputation":"unknown","message_encrypted":"False","message_signed":"False","nb_external_hops":"1","spam_method":"level","spam_score":"40.0","transport_signed":"True"}
,"pi":{"comportment":0,"context":0,"date_update":"0001-01-01T00:00:00Z","technic":0,"version":0}
,"raw_msg_id":"59fc2e24-5ebc-41f0-810c-055408cf84e3"
,"subject":"Avec ma nouvelle caisse enregistreuse l'encaissement est un jeu\n d'enfant !"
,"tags":null
,"type":"email"
,"user_id":"31141492-8116-479b-9b99-631d77f223bc"
}]}
```

Response when updating message:
```
{"attachments":null
,"body":"\n\n\n\n\n\n\n\u003cp\u003e\n\u003ca href=\"http://app.trouver-presta.fr/v/?camp=9544767838567738696_8\u0026amp;ms=bGF1cmVudEBicmFpbnN0b3JtLmZy\" title=\"Si cet e-mail ne s\u0026#39;affiche pas correctement, suivez ce lien.\" rel=\"nofollow\"\u003eSi cet email ne s’affiche pas correctement, suivez ce lien.\u003c/a\u003e\u003c/p\u003e\u003ctable width=\"600\"\u003e     \u003ctbody\u003e         \u003ctr\u003e             \u003ctd width=\"600\" height=\"40\" align=\"center\" valign=\"middle\"\u003eMa Nouvelle Caisse enregistreuse est un jeu d\u0026#39;enfant.\u003cbr\u003e             Si ce message ne s\u0026#39;affiche pas correctement, cliquez ici.\u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd width=\"600\" height=\"77\" align=\"center\"\u003e\u003cimg src=\"http://app.trouver-presta.fr/r/?rc=aHR0cHM6Ly9ub2Rlcy5uZW9wZXJmLmNvbS91cGxvYWRzL3Zpc3VlbHMvMTMvMjcva2l0cy9Mb2dvX0Jwcm8uanBn\" alt=\"BPRO\" width=\"136\" height=\"77\"\u003e\u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd width=\"600\" height=\"43\" align=\"center\" valign=\"middle\"\u003eCaisse enregistreuse\u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd width=\"600\" height=\"2\"\u003e \u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd\u003e             \u003ctable width=\"600\"\u003e                 \u003ctbody\u003e                     \u003ctr\u003e                         \u003ctd width=\"72\" height=\"15\"\u003e \u003c/td\u003e                         \u003ctd width=\"1\" height=\"15\"\u003e \u003c/td\u003e                         \u003ctd width=\"66\" height=\"15\"\u003e \u003c/td\u003e                         \u003ctd width=\"419\" height=\"15\"\u003e \u003c/td\u003e                         \u003ctd height=\"15\"\u003e \u003c/td\u003e                     \u003c/tr\u003e                 \u003c/tbody\u003e             \u003c/table\u003e             \u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd\u003e             \u003ctable width=\"600\"\u003e                 \u003ctbody\u003e                     \u003ctr\u003e                         \u003ctd\u003e\u003ca href=\"http://app.trouver-presta.fr/r/?m=bGF1cmVudEBicmFpbnN0b3JtLmZy\u0026amp;c=9544767838567738696_8\u0026amp;rc=aHR0cDovL25vZGVzLm5wdjE3cWVqYmQuY29tL3IvY2xpYy0yNy0zNjItMzAxOC11cmwtYUhSMGNEb3ZMMkp3Y204dVpuSXZNemN5TUY5Q1VETmZUbVZ2Y0Q5eFkzQTlNemN5TUY5Q1VETmZUbVZ2Y0NOMWRHMWZjMjkxY21ObFBXNG1kWFJ0WDIxbFpHbDFiVDFsTFcxaGFXd21kWFJ0WDJOaGJYQmhhV2R1UFc1bGQzTmpKblYwYlY5dWIyOTJaWEp5YVdSbFBURT9yZ3JvdXA9cF8xMTM1MSZhbXA7dHJhY2thZmY9JmFtcDtncm91cD0=\" rel=\"nofollow\"\u003e\u003cimg src=\"http://app.trouver-presta.fr/r/?rc=aHR0cHM6Ly9ub2Rlcy5uZW9wZXJmLmNvbS91cGxvYWRzL3Zpc3VlbHMvMTMvMjcva2l0cy9QaWN0b19FcXVpcG10LmpwZw==\" width=\"139\" height=\"98\" alt=\"\"\u003e\u003c/a\u003e\u003c/td\u003e                         \u003ctd\u003e       \n                   \u003ctable width=\"419\"\u003e                             \u003ctbody\u003e                                 \u003ctr\u003e                                     \u003ctd width=\"19\"\u003e \u003c/td\u003e                                     \u003ctd width=\"400\" height=\"4\"\u003e \u003c/td\u003e                                 \u003c/tr\u003e                                 \u003ctr\u003e                                     \u003ctd width=\"19\" height=\"60\"\u003e \u003c/td\u003e                                     \u003ctd width=\"400\" align=\"left\" valign=\"middle\"\u003e\u003cspan\u003eAvec ma nouvelle caisse enregistreuse\u003cbr\u003e                                     \u003c/span\u003el\u0026#39;encaissement est un jeu d\u0026#39;enfant !                 \u003c/td\u003e                                 \u003c/tr\u003e                                 \u003ctr\u003e                                     \u003ctd width=\"19\"\u003e \u003c/td\u003e                                     \u003ctd width=\"400\" height=\"4\"\u003e \u003c/td\u003e                                 \u003c/tr\u003e                             \u003c/tbody\u003e                         \u003c/table\u003e                         \u003c/td\u003e                         \u003ctd width=\"42\" height=\"98\"\u003e \u003c/td\u003e                     \u003c/tr\u003e                 \u003c/tbody\u003e             \u003c/table\u003e             \u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd\u003e             \u003ctable width=\"600\"\u003e           \n       \u003ctbody\u003e                     \u003ctr\u003e                         \u003ctd width=\"72\" height=\"53\"\u003e \u003c/td\u003e                         \u003ctd width=\"1\"\u003e \u003c/td\u003e                         \u003ctd width=\"66\"\u003e \u003c/td\u003e                         \u003ctd width=\"419\"\u003e\u003ca href=\"http://app.trouver-presta.fr/r/?m=bGF1cmVudEBicmFpbnN0b3JtLmZy\u0026amp;c=9544767838567738696_8\u0026amp;rc=aHR0cDovL25vZGVzLm5wdjE3cWVqYmQuY29tL3IvY2xpYy0yNy0zNjItMzAxOC11cmwtYUhSMGNEb3ZMMkp3Y204dVpuSXZNemN5TUY5Q1VETmZUbVZ2Y0Q5eFkzQTlNemN5TUY5Q1VETmZUbVZ2Y0NOMWRHMWZjMjkxY21ObFBXNG1kWFJ0WDIxbFpHbDFiVDFsTFcxaGFXd21kWFJ0WDJOaGJYQmhhV2R1UFc1bGQzTmpKblYwYlY5dWIyOTJaWEp5YVdSbFBURT9yZ3JvdXA9cF8xMTM1MSZhbXA7dHJhY2thZmY9JmFtcDtncm91cD0=\" rel=\"nofollow\"\u003e                         \u003cul\u003e                             \u003cli\u003eFacile\u003c/li\u003e                             \u003cli\u003ePratique\u003c/li\u003e                             \u003cli\u003eErgonomique\u003c/li\u003e   \n                       \u003c/ul\u003e                         \u003c/a\u003e\u003c/td\u003e                         \u003ctd width=\"42\"\u003e \u003c/td\u003e                     \u003c/tr\u003e                 \u003c/tbody\u003e             \u003c/table\u003e             \u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd\u003e             \u003ctable width=\"600\"\u003e                 \u003ctbody\u003e                     \u003ctr\u003e                         \u003ctd width=\"72\" height=\"16\"\u003e \u003c/td\u003e                         \u003ctd width=\"1\" height=\"16\"\u003e \u003c/td\u003e                                                  \u003ctd height=\"16\" width=\"485\"\u003e \u003c/td\u003e                                                  \u003ctd width=\"42\" height=\"16\"\u003e \u003c/td\u003e                     \u003c/tr\u003e                 \u003c/tbody\u003e             \u003c/table\u003e             \u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd\u003e             \u003ctable width=\"600\"\u003e                 \u003ctbody\u003e                     \u003ctr\u003e                         \u003ctd width=\"72\" height=\"178\"\u003e \u003c/td\u003e                         \u003ctd width=\"1\" height=\"178\"\u003e \u003c/td\u003e                         \u003ctd\u003e\u003ca href=\"http://app.trouver-presta.fr/r/?m=bGF1cmVudEBicmFpbnN0b3JtLmZy\u0026amp;c=9544767838567738696_8\u0026amp;rc=aHR0cDovL25vZGVzLm5wdjE3cWVqYmQuY29tL3IvY2xpYy0yNy0zNjItMzAxOC11cmwtYUhSMGNEb3ZMMkp3Y204dVpuSXZNemN5TUY5Q1VETmZUbVZ2Y0Q5eFkzQTlNemN5TUY5Q1VETmZUbVZ2Y0NOMWRHMWZjMjkxY21ObFBXNG1kWFJ0WDIxbFpHbDFiVDFsTFcxaGFXd21kWFJ0WDJOaGJYQmhhV2R1UFc1bGQzTmpKblYwYlY5dWIyOTJaWEp5YVdSbFBURT9yZ3JvdXA9cF8xMTM1MSZhbXA7dHJhY2thZmY9JmFtcDtncm91cD0=\" rel=\"nofollow\"\u003e\u003cimg src=\"http://app.trouver-presta.fr/r/?rc=aHR0cHM6Ly9ub2Rlcy5uZW9wZXJmLmNvbS91cGxvYWRzL3Zpc3VlbHMvMTMvMjcva2l0cy9WaXN1ZWwuanBn\" width=\"485\" height=\"178\" alt=\"\"\u003e\u003c/a\u003e\u003c/td\u003e                         \u003ctd width=\"42\" height=\"178\"\u003e \u003c/td\u003e                     \u003c/tr\u003e                 \u003c/tbody\u003e             \u003c/table\u003e                      \u003c/tr\u003e         \u003ctr\u003e             \u003ctd\u003e             \u003ctable width=\"600\"\u003e                 \u003ctbody\u003e                     \u003ctr\u003e                         \u003ctd width=\"72\" height=\"33\"\u003e \u003c/td\u003e                         \u003ctd width=\"1\" height=\"33\"\u003e \u003c/td\u003e                         \u003ctd width=\"527\" height=\"33\"\u003e \u003c/td\u003e                     \u003c/tr\u003e                 \u003c/tbody\u003e             \u003c/table\u003e             \u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd\u003e             \u003ctable width=\"600\"\u003e                 \u003ctbody\u003e                     \u003ctr\u003e                         \u003ctd width=\"72\" height=\"65\"\u003e \u003c/td\u003e                         \u003ctd width=\"1\" height=\"65\"\u003e \u003c/td\u003e                         \u003ctd width=\"91\" height=\"65\"\u003e \u003c/td\u003e                         \u003ctd align=\"center\"\u003e\u003ca href=\"http://app.trouver-presta.fr/r/?m=bGF1cmVudEBicmFpbnN0b3JtLmZy\u0026amp;c=9544767838567738696_8\u0026amp;rc=aHR0cDovL25vZGVzLm5wdjE3cWVqYmQuY29tL3IvY2xpYy0yNy0zNjItMzAxOC11cmwtYUhSMGNEb3ZMMkp3Y204dVpuSXZNemN5TUY5Q1VETmZUbVZ2Y0Q5eFkzQTlNemN5TUY5Q1VETmZUbVZ2Y0NOMWRHMWZjMjkxY21ObFBXNG1kWFJ0WDIxbFpHbDFiVDFsTFcxaGFXd21kWFJ0WDJOaGJYQmhhV2R1UFc1bGQzTmpKblYwYlY5dWIyOTJaWEp5YVdSbFBURT9yZ3JvdXA9cF8xMTM1MSZhbXA7dHJhY2thZmY9JmFtcDtncm91cD0=\" rel=\"nofollow\"\u003e\u003cimg src=\"http://app.trouver-presta.fr/r/?rc=aHR0cHM6Ly9ub2Rlcy5uZW9wZXJmLmNvbS91cGxvYWRzL3Zpc3VlbHMvMTMvMjcva2l0cy9CdG5fRXF1aXBtdC5qcGc=\" alt=\"en savoir plus\" width=\"272\" height=\"65\"\u003e\u003c/a\u003e\u003c/td\u003e                         \u003ctd width=\"164\" height=\"65\"\u003e \u003c/td\u003e                     \u003c/tr\u003e                 \u003c/tbody\u003e             \u003c/table\u003e             \u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd\u003e\u003cimg width=\"600\" height=\"33\" alt=\"\"\u003e\u003c/td\u003e         \u003c/tr\u003e         \u003ctr\u003e             \u003ctd width=\"600\" height=\"35\" align=\"center\"\u003e \u003c/td\u003e         \u003c/tr\u003e     \u003c/tbody\u003e \u003c/table\u003e\u003cimg alt=\"\" src=\"http://app.trouver-presta.fr/r/?rc=aHR0cDovL25vZGVzLm5wdjE3cWVqYmQuY29tL21haWxpbmctMjctMzYyLTMwMTg/cmdyb3VwPXBfMTEzNTEmYW1wO2dyb3VwPQ==\" height=\"1\" width=\"1\"\u003e\u003cimg src=\"http://app.trouver-presta.fr/t/?i=9544767838567738696_8\u0026amp;m=bGF1cmVudEBicmFpbnN0b3JtLmZy\u0026amp;url=http://app.trouver-presta.fr/images/blank.jpg\" alt=\"_pspacer5\"/\u003e\u003cp\u003e\n\u003ca href=\"http://app.trouver-presta.fr/d/?camp=9544767838567738696_8\u0026amp;ms=bGF1cmVudEBicmFpbnN0b3JtLmZy\" title=\"Ne plus recevoir d\u0026#39;informations de notre part\" rel=\"nofollow\"\u003ePour se désabonner : Suivez ce lien.\u003c/a\u003e\u003cbr/\u003eSi ce message vous a causé un quelconque dérangement, nous vous prions de nous en excuser.\n\u003c/p\u003e\n\n\n\n\n"
,"body_is_plain":false,"excerpt":"Ma Nouvelle Caisse enregistreuse est un jeu d'enfant. Si ce message ne s'affiche pas correctement, cliquez ici. Caisse enregistreuse Avec ma nouvelle caisse enregistreuse l'encaissement est un jeu …"
,"date":"2016-04-05T03:56:40.000Z","date_delete":"0001-01-01T00:00:00.000Z","date_insert":"2017-09-28T07:21:02.666Z","discussion_id":"15c5f51c-027d-4852-9eb0-f52eed7512b8"
,"external_references":{"ancestors_ids":[],"message_id":"7af37de14807c55ac99890f58e2d2aca@trouver-presta.fr","parent_id":""}
,"identities":null
,"importance_level":-2
,"is_answered":false
,"is_draft":false
,"is_unread":true
,"message_id":"732156e2-e969-4259-ac1e-831d7c8e32b9"
,"parent_id":""
,"participants":[{"address":"caisse@trouver-presta.fr","contact_ids":["d46a8028-13fa-4eb8-9f96-24157bc4413b"],"label":"caisse@trouver-presta.fr","protocol":"email","type":"From"},{"address":"dev@caliopen.local","contact_ids":["abfa55df-9a47-4107-8a24-63752d8b7fd8"],"label":"dev@caliopen.local","protocol":"email","type":"To"}]
,"privacy_features":null
,"pi":{"comportment":0,"context":10,"date_update":"0001-01-01T00:00:00Z","technic":10,"version":0}
,"raw_msg_id":"59fc2e24-5ebc-41f0-810c-055408cf84e3"
,"subject":"Avec ma nouvelle caisse enregistreuse l'encaissement est un jeu\n d'enfant !"
,"tags":null
,"type":"email"
,"user_id":"31141492-8116-479b-9b99-631d77f223bc"
}
```

**Comment by gdchamal** [comment]
The /discussions route is deprecated and is not able to give correct values for PI.
This is "normal" to have only 0 values on this route.

The correct PI value is on /messages route

**Comment by iamdey** [comment]
@gdchamal the route `/discussion` @myleneb is talking about is form the client (not API)

we should make sure that there's no differences in messages entities between
`api/v/messages/?<whatever>` and `api/v/messages/<id>`

**Comment by sapiens-sapide** [comment]
I'm taking over this issue as I can reproduce it. It's a bug in the API.

**Comment by sapiens-sapide** [comment]
Fix by commit #fa8e70b
